backend/src/lib.rs

```diff
@@ -61,9 +61,16 @@ impl LTN {
         });
 
         let boundary: Feature = serde_wasm_bindgen::from_value(boundary_input)?;
-        let polygon = boundary.try_into().map_err(err_to_js)?;
-
-        let map = MapModel::new(input_bytes, polygon, study_area_name).map_err(err_to_js)?;
+        let boundary_geom: geo::Geometry = boundary.try_into().map_err(err_to_js)?;
```

Comment:
Is there any more shorthand way to deserialize a GJ polygon or multipolygon, and `into()` it into a `geo::MultiPolygon` that you know of?

(Also, I didn't describe testing in the PR description -- I did verify new areas with islands and old one-polygon areas both work)